Tools/scripts/deepfreeze.py

```diff
@@ -440,6 +442,18 @@ def generate(args: list[str], output: TextIO) -> None:
             else:
                 code = compile(fd.read(), f"<frozen {modname}>", "exec")
             printer.generate_file(modname, code)
+    with printer.block("static void\ndealloc_codeobject(PyCodeObject *co, _Py_CODEUNIT *firstinstr)"):
```

Comment:
What about putting this code in Objects/codeobject.c? I suggest the name _PyStaticCode_Dealloc(). You can declare the function in Include/internal/pycore_code.h.

Reply:
I have some thoughts about this:

- This function is only usable in deepfreeze as it needs to restore the instructions pointer of codeobject which is unlikely in other scenario.
- The python core currently does not have any immortal objects support so it isn't usable in there too.
- That can be easily moved when there is more need to such function as currently as I don't think there is any so I think this is good enough for now.